src/main/java/dk/kb/image/TemplateMostUsedColors.java

```diff
@@ -0,0 +1,136 @@
+package dk.kb.image;
+
+import dk.kb.image.model.v1.DominantColorDto;
+
+import java.awt.image.BufferedImage;
+import java.util.*;
+import java.util.stream.Collectors;
+
+/**
+ * Template to extract dominant colors from image.
+ * Implementations need to implement a colorspace to work with.
+ * @param <C> represents the datatype that the specific implementation implements.
+ */
+abstract class TemplateMostUsedColors<C> {
+    int pixelCount = 0;
+
+    /**
+     * Template for color analysis. The template defines how the algorithm runs and the subclasses/implementations defines which colorspace to use.
```

Comment:
Shouldn't this line be in the JavaDoc for the class itself? It looks to be a general description rather than a method description.